src/lib/components/data-table/DataTable.svelte

```
@@ -0,0 +1,177 @@
+							title={expandable && index === 0 && row.getCanExpand() ? 'View options' : undefined}
+						>
+							<FlexRender content={cell.column.columnDef.cell} context={cell.getContext()} />
+						</Table.Cell>
+					{/each}
+				</Table.Row>
+
+				{#if expandable && row.getIsExpanded() && renderExpandedRow}
+					<Table.Row class="border-t-0">
+						<Table.Cell colspan={columns.length} class="p-0">
+							<div class="border-t p-4">
+								{@html renderExpandedRow(row.original)}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)